### PR TITLE
[Runtime-1059] Collection runner: Describe customizing requests and saving cookies

### PR DESCRIPTION
**Pull request by arnitkun**
Added the following pieces of information to "starting a collection run" section:

1. Customising the execution of requests, specifically, the order and the state of execution i.e. if a request will be executed or not.

2. Saving the cookies for after the collection run to the collection manager.

### CODE REVIEW

File: v6/postman/collection_runs/starting_a_collection_run.md [hunk 1]
```diff
@@ -68,11 +78,19 @@ Supplies a data file to use for the collection run.
 To learn more about how to use a date file for a collection run, see
 [Working with data files](/docs/v6/postman/collection_runs/working_with_data_files).
 
-## Persist variables
+## Keep variable values
```
Comment (kaustavdm) [inline v6/postman/collection_runs/starting_a_collection_run.md]: Changing this heading might break incoming links. Anchor tags are generated based on heading texts. Change these back to persist variables.

File: v6/postman/collection_runs/starting_a_collection_run.md [hunk 2]
```diff
@@ -9,6 +9,14 @@ Collections are groups of requests that can be run together as a series of reque
 
 You can use scripts to build integration test suites, pass data between API requests, and build workflows that mirror your actual use case of APIs.
 
+## Customising requests in collection runner
```
Comment (kaustavdm) [inline v6/postman/collection_runs/starting_a_collection_run.md]: "Customizing". Use US English.

File: v6/postman/collection_runs/starting_a_collection_run.md (hunk 1 above)
Comment (arnitkun) [reply v6/postman/collection_runs/starting_a_collection_run.md]: Reverted to the original name.

File: v6/postman/collection_runs/starting_a_collection_run.md (hunk 2 above)
Comment (arnitkun) [reply v6/postman/collection_runs/starting_a_collection_run.md]: Done.

File: v6/postman/collection_runs/starting_a_collection_run.md [hunk 3]
```diff
@@ -9,6 +9,14 @@ Collections are groups of requests that can be run together as a series of reque
 
 You can use scripts to build integration test suites, pass data between API requests, and build workflows that mirror your actual use case of APIs.
 
+## Customizing requests in collection runner
+
+The collection runner allows for customising the execution of requests in the following ways:
```
Comment (kaustavdm) [inline v6/postman/collection_runs/starting_a_collection_run.md]: "customizing"

File: v6/postman/collection_runs/starting_a_collection_run.md [hunk 4]
```diff
@@ -70,9 +80,17 @@ To learn more about how to use a date file for a collection run, see
...
+To learn more about Keep variable values in a collection run, see [Using environments in collection runs](/docs/v6/postman/collection_runs/using_environments_in_collection_runs).
+
+## Run collection without using stored cookies
+
+Enabling this allows collection runs without using cookies stored in the cookie manager.
+
+## Save cookies after collection run
 
-To learn more about persist variables in a collection run, see [Using environments in collection runs](/docs/v6/postman/collection_runs/using_environments_in_collection_runs).
+Enabling this saves the cookies for the session to the cookie manager.
```
Comment (kaustavdm) [inline v6/postman/collection_runs/starting_a_collection_run.md]: Add some more description around what this is supposed to do for Collection Runner. How does this selection affect runner behaviour and when should someone use this option?

File: v6/postman/collection_runs/starting_a_collection_run.md [hunk 5]
```diff
@@ -70,9 +80,17 @@ To learn more about how to use a date file for a collection run, see
 
 ## Persist variables
 
-By default, any environment changes in the collection runner are not reflected in the request builder.
+By default, any environment changes in the collection runner are not reflected in the request builder. Enabling this option saves the values of the variables at the end of the run.
+
+To learn more about Keep variable values in a collection run, see [Using environments in collection runs](/docs/v6/postman/collection_runs/using_environments_in_collection_runs).
+
+## Run collection without using stored cookies
+
```
Comment (kaustavdm) [inline v6/postman/collection_runs/starting_a_collection_run.md]: Add more details around when should someone use this.

File: v6/postman/collection_runs/starting_a_collection_run.md (hunk 4 above)
Comment (arnitkun) [reply v6/postman/collection_runs/starting_a_collection_run.md]: Added additional information.

File: v6/postman/collection_runs/starting_a_collection_run.md (hunk 5 above)
Comment (arnitkun) [reply v6/postman/collection_runs/starting_a_collection_run.md]: Added additional information.

File: v6/postman/collection_runs/starting_a_collection_run.md (hunk 3 above)
Comment (kaustavdm) [reply v6/postman/collection_runs/starting_a_collection_run.md]: @arnitkun This spelling issue is still there.

Comment (arnitkun) [reply v6/postman/collection_runs/starting_a_collection_run.md]: fixed it.

**Pull request merged**